tests/login.test.js

```diff
@@ -19,5 +19,4 @@ describe('log in to the admin page', () => {
       .then(() => browser.getText('h1.page-header'))
       .then(header => expect(header).to.equal('Overview'));
   });
-
 });
```

Comment:
* Any reason this is not using the common login function?
* Potentially we should probably look into LDAP authentication as well, so not just local
* Test readonly, etc.